app/utils/xbrl_processing.py

```diff
@@ -78,7 +80,7 @@ def create_viewer_html(output_file : str,
     script_tag = soup.find('script', {'src': 'ixbrlviewer.js'})
     if script_tag:
         # Update the src attribute
-        script_tag['src'] = '{{ url_for(\'static\', filename=\'js/ixbrlviewer.js\') }}'
+        script_tag['src'] = '/static/js/ixbrlviewer.js'
```

Comment:
@lucakato 

2 thoughts here:

1. Can you confirm that the `ixbrlviewer.js` is exactly the same even with different Excel inputs? I suspect that it is not. (This script is generated when we run the command line arelle command `CntlrCmdLine.parseAndRun(args)` ). If it's different, we'll need to save the script inside the session-specific file and then refer to that file path.

2. Looking at this again, I think the issue is that we're inserting `'{{ url_for(\'static\', filename=\'js/ixbrlviewer.js\') }}'` as a string into the `viewer.html`. We should be able to edit 
```
script_tag['src'] = '{{ url_for(\'static\', filename=\'js/ixbrlviewer.js\') }}'
```
to 
```
script_tag['src'] = url_for(\'static\', filename=\'js/ixbrlviewer.js\')
```
which still inserts the absolute url into the html, but keeps the python flexible in case the file structure changes when we move to AWS.

Reply:
@kwheelan Sorry for the late reply.
Regarding 1) when I tested it with `script_tag['src'] = '/static/js/ixbrlviewer.js'` , the ixbrlviewer.js is the same/never changes.

2) did work too, I didn't see any errors. I'm going to work on getting the output.html again and check again.